examples/HEMS/utils/scheduling_utils.py

```diff
@@ -0,0 +1,36 @@
+from typing import Any
+
+from const import EV_CONFIG
+
+
+def create_dynamic_storage_flex_model(
+    current_soc: float,
+    constraints: dict = None,
+) -> dict[str, Any]:
+    """
+    Create a dynamic flex model for scheduling storage devices.
+
+    This function builds only the *ad hoc* part of the flex model, temporary values
+    defined at scheduling time for the current context, such as current SoC, SoC minima,
+    and SoC usage. Permanent properties of the device (e.g. capacities or efficiencies)
+    are defined on the asset's flex_model field.
```

Comment:
They "are defined on the asset" only in the tutorial - other users of this util function do not necessarily have that.
We should adapt this docstring so it fits more for general usage (and other docstrings in util functions, as well)